src/promptflow/promptflow/_utils/multimedia_utils.py

```diff
@@ -193,3 +195,25 @@ def recursive_process(value: Any, process_funcs: dict[type, Callable] = None) ->
     if isinstance(value, dict):
         return {k: recursive_process(v, process_funcs) for k, v in value.items()}
     return value
+
+
+def load_multimedia_data(inputs: dict[str, FlowInputDefinition], line_inputs: dict, base_dir: Path):
```

Comment:
May be can remove base_dir.
We should be loading from either absolute path or relative path to current working dir.
In other words, the path should be always readable without modifying.

Reply:
1. Good point, will remove in new PR.
2. It is not a class method, you can call it directly in flow executor.

Reply:
Ok, I see